masu/listener/run.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Masu listener entry-point."""
+import logging
+
+from masu import create_app
+from masu.celery import celery, update_celery_config
+from masu.config import Config
+from masu.external.kafka_msg_handler import initialize_kafka_handler
+from masu.prometheus_stats import initialize_prometheus_exporter
+
+
+def get_masu_log_level(masu_level):
```

Comment:
Can you help me understand why this function is needed?

Where is masu's logging level *NOT* a direct mapping of standard logging levels?  If there's no case where we want INFO to mean DEBUG, then I'm not sure i see why this mapping is necessary.

Reply:
Good catch.  It's no longer needed.  Before I integrated the listener with celery (creating the Masu app) I needed to setup logging and this was necessary.  I'l remove it.